pytorch_toolkit/tests/run_model_templates_tests.py

```diff
@@ -22,18 +23,55 @@
 
 ENABLE_TRAIN_TESTS = True
 ENABLE_EXPORT_TESTS = True
+ENABLE_NNCF_TESTS = True
 
+def _is_verbose_flag_set():
+    if '-v' in sys.argv or '--verbose' in sys.argv:
+        return True
+    else:
+        return False
+
+def run_with_log(*args, **kwargs):
+    cmd = args[0]
+    if _is_verbose_flag_set():
+        logging.info(f'Running command\n`{cmd}`') #TODO: consider with Ilya
```

Comment:
you can leave it without checking any conditions

Reply:
Ok

Reply:
Done